app/controllers/sign_up/personal_keys_controller.rb

```diff
@@ -1,14 +1,14 @@
 module SignUp
-  class RecoveryCodesController < ApplicationController
-    include RecoveryCodeConcern
+  class PersonalKeysController < ApplicationController
+    include PersonalKeyConcern
 
     before_action :confirm_two_factor_authenticated
-    before_action :confirm_has_not_already_viewed_recovery_code, only: [:show]
+    before_action :confirm_has_not_already_viewed_personal_key, only: [:show]
 
     def show
-      user_session.delete(:first_time_recovery_code_view)
+      user_session.delete(:first_time_personal_key_view)
       @code = new_code
-      analytics.track_event(Analytics::USER_REGISTRATION_RECOVERY_CODE_VISIT)
+      analytics.track_event(Analytics::USER_REGISTRATION_PERSONAL_KEY_VISIT)
     end
```

Comment:
Oh. I see what's happening. The tests corresponding to these changes haven't changed, so they aren't part of this PR, so CC thinks they're not tested. That's my guess.